tests/onboarding/test_onboarding_syncing.py

```diff
@@ -71,7 +71,8 @@ def test_sync_device_during_onboarding(multiple_instances):
 
         with step('Paste sync code on second instance and wait until device is synced'):
             sync_start = sync_view.open_enter_sync_code_form()
-            sync_start.paste_sync_code()
+            pyperclip.copy(sync_code)
```

Comment:
@Valentina1133 perhaps pyperclip is not great and sometimes the code value is not pasted. I know we return `pyperclip.paste()` when getting sync code, but i suspect we lose it somehow during the test run. So i just added one more force copying the code and it looks like it helps. Let me know what you think

Reply:
I ran the tests locally on linux and launched several manual jobs and it looks okay

Reply:
Great that it helped! I don't have any other idea